setup.py

```diff
@@ -60,56 +60,6 @@
 """
 
 from setuptools import setup
-from setuptools import find_packages
-
-
-with open('README.rst') as file_object:
-  long_description = file_object.read()
-
-setup(
-  name = 'securesystemslib',
-  version = '0.18.0',
```

Comment:
I tested locally with python 3.7 generating a source dist as it is still listed in the documentation:
` $ python setup.py sdist` 
The resulting archive is missing a version number: securesystemslib-**0.0.0**.  I assume this may be some incompatibility with `setuptools_scm`?

In case I was not supposed to use `setup.py` to generate the sdist, what is the intended way of archive generation after this set of changes? Using the `pep517` module as shown in the PR description did the job but the package [documentation](https://pypi.org/project/pep517/) marks its high-level functionality as deprecated.
Seems like the recommended way is by using [build](https://pypa-build.readthedocs.io/en/latest/).

Reply:
Try to upgrade setuptools. You need a recent (at least January 2019) setuptools to understand pyproject.toml, and if it understands it, it would upgrade itself to the version having the needed hooks `setuptools_scm` that uses `pyproject.toml` for config depends on.

>Seems like the recommended way is by using `build`.

Yes, it's true. The PR was originally created when there was no `build`.